### PR TITLE
Add Oregon state uprating

### LINKED ISSUE #4499: Add Oregon state uprating

### PR DESCRIPTION
**Pull request by Kiki-67**
Fixes #4499

### CODE REVIEW

File: policyengine_us/parameters/gov/states/or/tax/income/credits/exemption/amount.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/or/tax/income/credits/exemption/amount.yaml:1]: need chnangelog

File: policyengine_us/parameters/gov/states/or/tax/income/credits/retirement_income/income_threshold.yaml [hunk 1]
```diff
@@ -15,17 +15,32 @@ metadata:
 
 
 JOINT:
-  2018-01-01: 30_000
-  2023-01-01: 45_000
+  values:
+    2018-01-01: 30_000
+    2023-01-01: 45_000
+  metadata:
+    uprating: gov.irs.uprating
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/or/tax/income/credits/retirement_income/income_threshold.yaml:22]: this seems to be rounded

File: policyengine_us/parameters/gov/states/or/tax/income/credits/kicker/percent.yaml [hunk 2]
```diff
@@ -6,7 +6,8 @@ values:
 metadata:
   unit: /1
   period: year  
-  label: Oregon kicker rate  
+  label: Oregon kicker rate
+  uprating: gov.irs.uprating
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/or/tax/income/credits/kicker/percent.yaml:10]: I dont think this is uprated with inflation -> not every parameter should be please check the legal code for the relevant sections

File: policyengine_us/parameters/gov/states/or/tax/income/deductions/standard/amount.yaml [hunk 3]
```diff
@@ -17,46 +17,61 @@ metadata:
...
+    2003-01-01: 3_280
+    # Rose with inflation, not all dates documented.
+    2018-01-01: 4_435
+    2019-01-01: 4_545
+    2020-01-01: 4_630
+    2021-01-01: 4_700
+    2022-01-01: 4_840
+    2023-01-01: 5_210
+  metadata:
+    uprating: gov.irs.uprating
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/or/tax/income/deductions/standard/amount.yaml:30]: rounding?

File: policyengine_us/parameters/gov/states/or/tax/income/deductions/standard/claimable_as_dependent/min.yaml [hunk 4]
```diff
@@ -7,6 +7,7 @@ metadata:
   unit: currency-USD
   period: year
   label: Oregon minimum deduction if claimable as dependent
+  uprating: gov.irs.uprating
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/or/tax/income/deductions/standard/claimable_as_dependent/min.yaml:10]: rounded -> all parameters that seem to be adjusted to a round number should be checked for rounding

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

File: policyengine_us/parameters/gov/states/or/tax/income/credits/kicker/percent.yaml [hunk 5]
```diff
@@ -6,7 +6,7 @@ values:
 metadata:
   unit: /1
   period: year  
-  label: Oregon kicker rate  
+  label: Oregon kicker rate
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/or/tax/income/credits/kicker/percent.yaml:9]: I dont think this should be uprated

File: policyengine_us/parameters/gov/states/or/tax/income/credits/retirement_income/income_threshold.yaml [hunk 6]
```diff
@@ -15,17 +15,47 @@ metadata:
...
-  2018-01-01: 15_000
-  2023-01-01: 22_500
+  values:
+    2018-01-01: 15_000
+    2023-01-01: 22_500
+  metadata:
+    uprating: gov.irs.uprating
+    rounding:
+        type: downwards
+        interval: 100
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/or/tax/income/credits/retirement_income/income_threshold.yaml:34]: are you sure that the interval should be 100 and not 500

File: policyengine_us/parameters/gov/states/or/tax/income/rates/head_of_household.yaml [hunk 7]
```diff
@@ -33,6 +33,8 @@ brackets:
       2021-01-01: 7_300
       2022-01-01: 7_500
       2023-01-01: 8_100
+    metadata:
+      uprating: gov.irs.uprating
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/or/tax/income/rates/head_of_household.yaml:37]: rounding?

File: policyengine_us/parameters/gov/states/or/tax/income/rates/joint.yaml [hunk 8]
```diff
@@ -33,6 +33,8 @@ brackets:
       2021-01-01: 7_300
       2022-01-01: 7_500
       2023-01-01: 8_100
+    metadata:
+      uprating: gov.irs.uprating
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/or/tax/income/rates/joint.yaml:37]: rounding?

File: policyengine_us/parameters/gov/states/or/tax/income/rates/separate.yaml [hunk 9]
```diff
@@ -38,6 +38,8 @@ brackets:
       2021-01-01: 3_650
       2022-01-01: 3_750
       2023-01-01: 4_050
+    metadata:
+      uprating: gov.irs.uprating
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/or/tax/income/rates/separate.yaml:42]: rounding?

File: policyengine_us/parameters/gov/states/or/tax/income/rates/single.yaml [hunk 10]
```diff
@@ -38,6 +38,8 @@ brackets:
       2021-01-01: 3_650
       2022-01-01: 3_750
       2023-01-01: 4_050
+    metadata:
+      uprating: gov.irs.uprating
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/or/tax/income/rates/single.yaml:42]: rounding?

File: policyengine_us/parameters/gov/states/or/tax/income/rates/widow.yaml [hunk 11]
```diff
@@ -33,6 +33,8 @@ brackets:
       2021-01-01: 7_300
       2022-01-01: 7_500
       2023-01-01: 8_100
+    metadata:
+      uprating: gov.irs.uprating
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/or/tax/income/rates/widow.yaml:37]: rounding?

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

File: policyengine_us/parameters/gov/states/or/tax/income/credits/kicker/percent.yaml (hunk 5 above)
Comment (Kiki-67) [reply policyengine_us/parameters/gov/states/or/tax/income/credits/kicker/percent.yaml:9]: delete the uprating in amount.yaml file

File: policyengine_us/parameters/gov/states/or/tax/income/credits/retirement_income/income_threshold.yaml [hunk 12]
```diff
@@ -15,17 +15,47 @@ metadata:
...
+    rounding:
+        type: downwards
+        interval: 500
 HEAD_OF_HOUSEHOLD:
-  2018-01-01: 15_000
-  2023-01-01: 22_500
+  values:
+    2018-01-01: 15_000
+    2023-01-01: 22_500
+  metadata:
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/or/tax/income/credits/retirement_income/income_threshold.yaml:30]: could we double check if we need to uprate - I dont think we do

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

**Review by PavelMakarchuk — APPROVED** [review]

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**